src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -47,10 +50,35 @@
 #include "ui_automatedrssdownloader.h"
 #include "automatedrssdownloader.h"
 
+class AutomatedRssDownloader::DownloadRuleListMatchState
```

Comment:
It's more useful for PR #6178 (200a815) which has to hold a lot more state. By including it in these commits it makes for more straightforward diffs IMO. But I'm happy to move them to `AutomatedRssDownloader` if you'd prefer.